src/libcommonserver/utility/utility_win.cpp

```diff
@@ -112,15 +112,50 @@ static bool initTrusteeWithUserSID() {
 }
 
 static bool init_private() {
+    const std::string useGetRightsFallbackMethod = CommonUtility::envVarValue("KDRIVE_USE_GETRIGHTS_FALLBACK_METHOD");
+    if (!useGetRightsFallbackMethod.empty()) {
+        LOG_DEBUG(Log::instance()->getLogger(), "KDRIVE_USE_GETRIGHTS_FALLBACK_METHOD env is set, using fallback method");
+        IoHelper::_getAndSetRightsMethod = 1;
```

Comment:
It is surprising to see a private variable of the class `IoHelper` to be set in the body of a private function of `KDC::Utility`.

We can use instead a public setter or do this initialization within the `IoHelper` class.

Reply:
I moved the initialization from utility to IoHelper. it was an artefacts from before the creation of the IoHelper class.

Reply:
[43d8601](https://github.com/Infomaniak/desktop-kDrive/pull/155/commits/43d8601f32d5a94a22f1da7e4fcab280de83f82a)